pkg/pgmodel/migrate.go

```diff
@@ -123,16 +145,6 @@ func Migrate(db *pgx.Conn, versionInfo VersionInfo) (err error) {
 		return fmt.Errorf("Error encountered during migration: %w", err)
 	}
 
-	ExtensionIsInstalled = false
-	err = migrateExtension(db, "promscale", extSchema, version.ExtVersionRange, version.ExtVersionRangeString)
-	if err != nil {
-		log.Warn("msg", fmt.Sprintf("could not install promscale: %v. continuing without extension", err))
-	}
-
-	if err = checkExtensionsVersion(db); err != nil {
-		return fmt.Errorf("Error encountered while migrating extension: %w", err)
-	}
-
 	metadataUpdate(db, ExtensionIsInstalled, "version", versionInfo.Version)
```

Comment:
This line requires ExtensionIsInstalled to be set correctly. Right now, it's only set after this function returns, when `InstallUpgradePromscaleExtensions` is called. Which won't work. Maybe this should be split out into a separate function called from CreateClient or just move the `InstallUpgradePromscaleExtensions` call back in here ?